src/sentry/utils/email.py

```diff
@@ -342,6 +348,11 @@ def get_built_messages(self, to=None, bcc=None):
             logger.debug('Did not build any messages, no users to send to.')
         return results
 
+    def format_to(self, to):
+        if to and len(to) > MAX_RECIPIENTS:
```

Comment:
There are some notification emails being built in the test suite that don't get users attached, so `to` turns into a `NoneType`